golden-set.md

```diff
@@ -0,0 +1,93 @@
+# [Road Regulations Pdf](https://www.guadalupe.gob.mx/nuevo-reglamento-de-transito-homologado/)
+
+
+## Query: Mi luz de atrás se acaba de romper
+## Output: 166, 24, 152
+* Luz:
+  * 4 - 2
+  * 10 - 1
+  * 24 - 16
+  * 29 - 1
+  * 32 - 1
+  * 51 - 1
+  * 82 - 2
+  * 87 - 1
+  * 102 - 2
+  * 105 - 1
+  * 133 - 1
+  * 152 - 10
+  * 166 - 37
+* Atrás
+  * 24 - 5
+  * 51 - 1
+  * 89 - 1
+  * 90 - 1
+  * 122 - 1
+  * 137 - 1
+  * 150 - 2
+  * 152 - 1
+  * 166 - 10
+  * Anexo 5 - 1
+* Acaba
+  * None
+* Romper
+  * None
+
+## Query: Choque
+## Output: 137, 166, Anexo 5
+* 137 - 5
+* 166 - 5
+* Anexo 5 - 1
+
+## Query: ¿Me pueden multar por un neumático ponchado?
+## Output: 24, 130, 136, 158, 166, 168, 169, 172, 173
+
+* Multa
+  * 24 - 1
+  * 130 - 1
+  * 136 - 1
+  * 158 - 1
+  * 166 - 4
+  * 168 - 1
+  * 169 - 3
+  * 172 - 1
+  * 173 - 1
+
+## Query: La carroceria
+## Output: 4, 24,  166
+
+* Carroceria
+  * 4 - 1
+  * 24 - 1
+  * 166 - 2
+
+## Query: Permiso de conducir
+## Output: 4, 24, 32, 43, 49, 51, 107, 108, 109, 110, 111, 112, 115, 117, 124, 126, 130, 131, 134, 166, 170, 171, 173, 175
```

Comment:
Is this output ordered by relevance? If so, I'm curious why 166 is not the first result.

Reply:
I think this question is for @oscardavidrm but it seems he ordered them ascendingly

Reply:
Yes, I just wrote them as I found them. There's no sorting in there.